Gemfile.save

```diff
@@ -0,0 +1,54 @@
+source 'https://rubygems.org'
+
+git_source(:github) do |repo_name|
+  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
+  "https://github.com/#{repo_name}.git"
+end
+
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'rails', '~> 5.1.1'
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3'
```

Comment:
We will use Postgresql

brew install postgresql
brew services postgresql start